synthetics-client/suites/sample_testsuite.yml

```diff
@@ -0,0 +1,45 @@
+---
+  name: "Sample Test Suite"
+  type: "http"
+  appId: ""
+  notebookId: ""
+  savedQueryId: ""
+  savedVisualizationId: ""
+  operationalPanelId: ""
+  enabled: true
+  ipv4: true
+  ipv6: true
+  resolverMode: "all"
+  timeoutSeconds: 16
+  tags:
+  - "news"
+  - "apis"
+  keep_null: true
+  hosts:
+  - "https://opensearch.org/"
+  - "https://opensearch.org/synthetics"
+  - "https://github.com/opensearch-project"
+  maxRedirects: 0
+  proxyURL: ""
+  proxyHeaders: {}
+  username: ""
+  password: ""
+  ssl: {}
+  indexHeaders: true
+  indexResponse: "always"
+  request:
+    method: "GET"
+    headers: {'Accept-Encoding': None, 'Content-Encoding':'gzip'}
+    body: ""
+    json: {}
+  response:
+    status:
+    - 200
+    - 301
+    headers: {}
+    body: {}
+  scheduler:
+    schedule-type: "interval"
```

Comment:
Since there can be many different kinds of user inputs, all of that is detailed separately as to not clog the sample test-suite file. I'm going to add a separate README.md file however that can also give more insight into user input.

Reply:
I still feel like it's better to add the README along with this PR to not confuse user. But I am OK if you want to add it later. Could you create an issue for tracking this?

Reply:
Just added